packages/react-composites/src/composites/common/VideoEffectsPane.tsx

```diff
@@ -134,6 +134,11 @@ const VideoEffectsPaneTrampoline = (
   const locale = useLocale();
   /* @conditional-compile-remove(video-background-effects) */
   const selectedEffect = useSelector(activeVideoBackgroundEffectSelector);
+  /* @conditional-compile-remove(video-background-effects) */
+  const headerStyles = {
+    zIndex: 0
```

Comment:
the default zindex for any element is `0`, what does setting this to `0` do?

Reply:
Oh this is the panel's z-index, seems fine to do this though I will be removing use of the panel when the converged panel work is complete

Reply:
Yes, the panel has Z-index:1 so by setting z-index for header to 0, tooltip can overlay the header.